src/pages/journal/components/empty-slot/empty-slot.scss

```diff
@@ -0,0 +1,47 @@
+empty-slot {
+  ion-card {
+    &.grey-bg {
+      background: transparent;
+    }
+    border: 4px dashed #4A4A4A;
+    ion-grid {
```

Comment:
Is there any way to remove the `ion-grid` style? I can't see it as part of your template, so I'm assuming it's part of the internals of one of the Ionic components.

Reply:
I based my sccs on another - i've removed it and see no discernible difference.